src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -714,7 +728,7 @@ else if (word.equals(Filter.CLOSE_PARENTHESIS)) {// end of selected options
                             }
                             break;
                         case 17:// in additional fields query find the table and the column
-                            names = word.split("\\.");
+                            String[] names = word.split("\\.");
```

Comment:
why not add "if (word.contains(".")) {" here as well?

Reply:
make a method and reuse, please (case 2 and case 17, case 19
only state is different